src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -189,6 +189,685 @@ If you rely on external tools for storing and visualizing your cluster's time-se
+{{site.data.alerts.end}}
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl -X POST http://localhost:8080/_status/critical_nodes
+~~~
+
+~~~ json
+{
+  "criticalNodes": [
+  ],
+  "report": {
+    "underReplicated": [
+    ],
+    "overReplicated": [
+    ],
+    "violatingConstraints": [
+      {
+        "rangeDescriptor": {
+          "rangeId": "89",
+          "startKey": "8okSYW1zdGVyZGFtAAE=",
+          "endKey": "8okSYW1zdGVyZGFtAAESszMzMzMzQAD/gAD/AP8A/wD/AP8A/yMAAQ==",
+          "internalReplicas": [
+            {
+              "nodeId": 8,
+              "storeId": 8,
+              "replicaId": 5,
+              "type": 0
+            },
+            {
+              "nodeId": 4,
+              "storeId": 4,
+              "replicaId": 7,
+              "type": 0
+            },
+            {
+              "nodeId": 3,
+              "storeId": 3,
+              "replicaId": 6,
+              "type": 0
+            }
+          ],
+          "nextReplicaId": 8,
+          "generation": "40",
+          "stickyBit": {
+            "wallTime": "0",
+            "logical": 0,
+            "synthetic": false
+          }
+        },
+        "config": {
+          "rangeMinBytes": "134217728",
+          "rangeMaxBytes": "536870912",
+          "gcPolicy": {
+            "ttlSeconds": 14400,
+            "protectionPolicies": [
+            ],
+            "ignoreStrictEnforcement": false
+          },
+          "globalReads": false,
+          "numReplicas": 3,
+          "numVoters": 0,
+          "constraints": [
+            {
+              "numReplicas": 0,
+              "constraints": [
+                {
+                  "type": 1,
+                  "key": "region",
+                  "value": "europe-west1"
+                }
+              ]
+            }
+          ],
+          "voterConstraints": [
+          ],
+          "leasePreferences": [
+          ],
+          "rangefeedEnabled": false,
+          "excludeDataFromBackup": false
+        }
+      },
+      ...
+    ],
+    "unavailable": [
+    ],
+    "unavailableNodeIds": [
+    ]
+  }
+}
+~~~
+
+##### Replication status - under-replicated ranges
+
+The following example assumes you are running a fresh, local [`cockroach demo`](cockroach-demo.html) cluster started using the following command:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach demo --geo-partitioned-replicas --insecure
+~~~
+
+By default, this geo-distributed demo cluster will not have any [under-replicated ranges](ui-replication-dashboard.html#under-replicated-ranges).
+
+To force it into a state where some [ranges](architecture/overview.html#architecture-range) are under-replicated, issue the following [`ALTER TABLE ... CONFIGURE ZONE`](alter-table.html#configure-zone) statement, which tells it to store 9 copies of each range underlying the `rides` table.
```

Comment:
Suggesting a word change to be less ... forceful here; feel free to use another one.

Also, it's not immediately intuitive (to me) why this makes ranges under-replicated. It might help to mention the current number of copies (3...?) from which the cluster has to up-replicate to 9. Sorry if I've totally misunderstood what's going on here.

```suggestion
To put the cluster into a state where some [ranges](architecture/overview.html#architecture-range) are under-replicated, issue the following [`ALTER TABLE ... CONFIGURE ZONE`](alter-table.html#configure-zone) statement, which tells it to store 9 copies of each range underlying the `rides` table.
```

Reply:
SGTM re: word change

I think it's either 3 or 5 in this cluster by default - no you've totally understood!

it only makes them that way temporarily while they up-replicate to 1 on each node

another scenario could have been: set `num_replicas` to 18 on a 9-node cluster, which constraint cannot physically be met